### PR TITLE
Fixes #12965 - provisioning and config templates are shown from os

### PR DESCRIPTION
**Pull request by orrabin**

### CODE REVIEW

**Comment by ohadlevy** [comment]
typo in commit msg

File: test/functional/api/v2/provisioning_templates_controller_test.rb [hunk 1]
```diff
@@ -89,4 +89,9 @@ class Api::V2::ProvisioningTemplatesControllerTest < ActionController::TestCase
                    :provisioning_template => {:name => ''} }
     assert_response :unprocessable_entity
   end
+
+  test "should show templates from os" do
```
Comment (domcleal) [inline test/functional/api/v2/provisioning_templates_controller_test.rb:93]: Could you also add this to the config_templates_controller_test.rb please?

Comment (orrabin) [inline test/functional/api/v2/provisioning_templates_controller_test.rb:93]: @domcleal I added the test.

**Comment by domcleal** [comment]
Merged as 3e4b2f7, thanks @orrabin.

**Pull request closed without merging**